src/EditorFeatures/Core/Implementation/Suggestions/SuggestedActionsSource.cs

```diff
@@ -719,18 +741,27 @@ private static SuggestedActionSetPriority GetSuggestedActionSetPriority(CodeActi
                     if (result.HasFix)
                     {
                         Logger.Log(FunctionId.SuggestedActions_HasSuggestedActionsAsync);
-                        return true;
+                        if (result.Diagnostic.Severity == DiagnosticSeverity.Error)
+                        {
+                            // Show the error lightbulb if we know there's an error fix availalbe
+                            return PredefinedSuggestedActionCategoryNames.ErrorFix;
```

Comment:
this seems wrong.  We're only getting the first diagnostic with a fix... but how do we know the first diagnostic is the one that is the most severe?  For example, we might have a hidden diagnostic saying "you can make this expression-bodied method into a block-bodied method".  If that's the first diagnostic we get back, we won't show the .ErrorFix category even though that might appear there.